### PR TITLE
{bio}[foss/2017b,intel/2017b] HTSeq v0.9.1, Pysam v0.14

### PR DESCRIPTION
**Pull request by murfreesboro**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/5d740673a6ac7ab904e2753b2dd19940 for a full test report.

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['af5bba775e3fb45ed4cde64c691ebef36b0bf7a86efd35c884ad0734c27ad485']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Pysam', '0.14', versionsuffix),
+    ('matplotlib', '2.1.0', versionsuffix),
+]
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb]: As this is a PythonPackage you should add 
download_dep_fail = True
as Travis says.

The same goes for all easyconfigs here that are PythonPackage based.

Comment (murfreesboro) [reply easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb]: Thank you @akesandgren . I have added the option to all the four eb files.

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**FAILED**
Build succeeded for 2 out of 6 (6 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/3229b260c68fc2fb2bf0f6370fff80ca for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 19 out of 23 (6 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/fca4e8ec25c97f1c9ffca7b96780cfbd for a full test report.

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+use_pip = True
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': ["bin/htseq-count", "bin/htseq-qa"],
+    'dirs': ["lib/python%(pyshortver)s/site-packages/%(name)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg"],
```
Comment (akesandgren) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb]: The directory name is %(name)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg-info
so either add "-info" at the end of that 'dirs' test
or we could test for just lib/python%(pyshortver)s/site-packages/%(name)s
which is also created.

The same for the other HTSeq easyconfigs.

Comment (boegel) [reply easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb]: Just checking for `lib/python%(pyshortver)s/site-packages` is enough imho...

Comment (murfreesboro) [reply easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb]: I see the point - thank you for the information @boegel and @akesandgren !

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f4fe4c8777116d307487c52aabd1d7bf for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 6 out of 8 (6 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/a8cdcd0d1ae9d744c1a433203b45e979 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/d323e0637b238d1ad8af5c780fe3336b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2042.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cdbd444d0e889fb87f69be378ac6d165 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node3135.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/810fda4e48759714a404d11eebd5db56 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2496.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2a9b6affe9aef6156c6f7232d902d572 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @murfreesboro!

**Pull request merged**